src/styles/modules/_m-dropdown.sass

```diff
@@ -23,6 +23,10 @@
       opacity: 1
       visibility: visible
 
+  +is(with-events)
+    .#{_b(dropdown) + _e(arrow)}
+      pointer-events: auto !important;
```

Comment:
As this is SASS, I believe you should lose the semicolon. Also, any chance you could find a way to select this without the important? If you're trying to alter the style of another element within this element, maybe adding `.#{$prefix-component}` above line 26 might help?

Reply:
You got me here, I'll remove the semicolon and important as well!